go.mod

```diff
@@ -54,90 +54,6 @@ require (
 	sigs.k8s.io/yaml v1.3.0
 )
 
-require (
-	cloud.google.com/go v0.93.3 // indirect
-	cloud.google.com/go/storage v1.10.0 // indirect
-	filippo.io/edwards25519 v1.0.0-beta.2 // indirect
-	github.com/Workiva/go-datastructures v1.0.53 // indirect
-	github.com/aws/aws-sdk-go v1.40.45 // indirect
```

Comment:
I'm not sure removing this is correct with the newest go.

Reply:
this is automatically done by running go mod tidy, not sure how to prevent it

Reply:
hmm, when I run go mod tidy on this branch, it adds them back.

```
$ go version
go version go1.17.1 darwin/amd64
```

Reply:
I updated go and it brought it back.